### ISSUE TITLE
Unquoted colours passed to functions/mixins are flagged as incorrect

### ISSUE THREAD

**Issue by nickdunn** [brigade/scss-lint#175]
It is valid Sass syntax to pass arguments unquoted, e.g.

```
color: my_colours(black);
```

However scss-lint will flag this as an issue, incorrectly thinking it's a colour. It should recognise this as a function argument and not apply a lint rule.

> Color `black` should be written in hexadecimal form as `#000`

**Comment by kevinmpowell** [comment]
I had a similar issue and used this workaround:

**before**
````
$colors: (
  brand-primary:   #5298F1,
  white:   #FFFFFF,
  black:   #000000,
  gray-darkest:    #454545,
  gray-darker:     #686868,
  gray-dark:       #848484,
  gray-light:      #A4A4A4,
  gray-lighter:    #F2F2F2,
  gray-lightest:   #F8F8F8,
  blue-dark:       #4379BD,
  blue:    #4D8AD6,
  blue-light:      #5298F1,
  green:   #5B8C42,
  green-light:     #82A827,
  text-default:    #454545,
  text-light:      #8D8D8D,
  text-lighter:    #999999
);
````

So my collisions with the linter were occurring on "black", "white", "blue", and "green". 
Since I already had things like "blue-light", I simply changed "blue" to "blue-default". They're still not semantic names, but when you're first building out a system you don't know what your border colors, link colors, etc. are, but you have already defined a palette. 

**after**
````
$colors: (
  brand-primary:   #5298F1,
  white-default:   #FFFFFF,
  black-default:   #000000,
  gray-darkest:    #454545,
  gray-darker:     #686868,
  gray-dark:       #848484,
  gray-light:      #A4A4A4,
  gray-lighter:    #F2F2F2,
  gray-lightest:   #F8F8F8,
  blue-dark:       #4379BD,
  blue-default:    #4D8AD6,
  blue-light:      #5298F1,
  green-default:   #5B8C42,
  green-light:     #82A827,
  text-default:    #454545,
  text-light:      #8D8D8D,
  text-lighter:    #999999
);
````
